README.md

```
@@ -1,299 +1,84 @@
-```bash
-$ ./bin/test-local
-```
-
-### Docker Usage
+**Security Note**: The CLI will warn if Jupyter runs without authentication. For local development only. Set `DEEPNOTE_JUPYTER_TOKEN` for shared environments.
 
-We use Docker to ensure reproducible environments due to Jupyter libraries' binary dependencies:
+## Contributing
 
-- `builder.Dockerfile`: Creates Python package bundles for different versions (3.9-3.12), generates kernel and server bundles, and packages the toolkit for distribution using Poetry.
+For more details on how to set up the local development environment and contribute, 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)